shapely/geometry/base.py

```diff
@@ -181,14 +204,23 @@ def svg(self, scale_factor=1.0, **kwargs):
         """Raises NotImplementedError"""
         raise NotImplementedError
 
+    def svg_path_element(self, stroke_scale_factor=1.0):
+        path = ET.fromstring(self.svg(stroke_scale_factor))
+
+        # svg is y-down, shapely is y-up
+        _, ymin, _, ymax = self.bounds
+        path.set('transform', f'matrix(1,0,0,-1,0,{ymax + ymin})')
+        return path
+
+    def svg_doc(self, width, height, xmin, ymin, xmax, ymax, stroke_scale_factor=1):
+        path = self.svg_path_element(stroke_scale_factor)
+        doc_element = svg_doc_element([path], width, height, xmin, ymin, xmax, ymax)
+        return ET.tostring(doc_element, encoding="unicode")
+
     def _repr_svg_(self):
         """SVG representation for iPython notebook"""
-        svg_top = (
-            '<svg xmlns="http://www.w3.org/2000/svg" '
-            'xmlns:xlink="http://www.w3.org/1999/xlink" '
-        )
         if self.is_empty:
-            return svg_top + "/>"
+            return ET.tostring(empty_svg_doc_element(), encoding="unicode")  
```

Comment:
Great! Seeing `+ "/>"` in code makes me nervous. Using a tested serializer is way better.